### DISCUSSION TITLE
Status of 802.3bt

### DISCUSSION THREAD

**Discussion by Ekmansoft** [lldpd/lldpd#697] [Q&A]
I see in the latest release from early this year that 802.3bt support seem to have been removed as it was broken?

Can you describe what was removed and what was the problem? Could we fix it?

**Comment by vincentbernat** [comment]
The explanation could is in https://github.com/lldpd/lldpd/pull/449#issuecomment-1763464634.

Commits removing code:
- https://github.com/lldpd/lldpd/commit/f01183f986bd31e4306a363280e0a1154955cf9d
- https://github.com/lldpd/lldpd/commit/3d18520ece5d4937ad398bd0a5f40331059ab456

**Reply by Ekmansoft** [reply]
Now we have tested with some 802.3bt switches, using the 1.0.17 version, where the 802.3bt send power-via-MDI code for 29-byte TLVs was still available. 

It works fine as far as we can see. We would like to have it latest master again. 
What kind of problems had been reported?

**Reply by Ekmansoft** [reply]
@vincentbernat please let us know what problems you have seen (or heard of) regarding the 802.3bt power-via-MDI 29-byte power request . 

We would like to create a new patch (and pull request) to add the power request functionality to lldpd again, or if you do not intend to add it again, we must consider whether we want to move it to become a local patch instead.

**Reply by vincentbernat** [reply]
Could you capture the packets produced in a pcap? In the comment, I say that Wireshark complains, let's check if this is the case.

**Reply by Ekmansoft** [reply]
I can tell you that wireshark does not complain anymore, but I did need to update wireshark last year, because the older version of the decoder did not support one change done between 802.3at and bt. I will send a pcap file later.

**Reply by vincentbernat** [reply]
OK, I may have been too fast to remove support. I don't remember exactly. We can restore it then.

**Comment by Ekmansoft** [comment]
[simple-802-3-setup-2.zip](https://github.com/user-attachments/files/29003168/simple-802-3-setup-2.zip)

Here is a simple pcap file with 29-byte power-via-MDI packets between a Netgear MS108TUP and one of our devices.
